shared/fixes/bash/file_owner_etc_shadow.sh

```diff
@@ -0,0 +1,2 @@
+# platform = multi_platform_all
+chown root /etc/shadow
```

Comment:
Not required, shared/templates/create_file_owner.py should generate this fix for you from template